policyengine-client/src/countries/uk/data/situation.jsx

```diff
@@ -68,10 +83,25 @@ export const VARIABLE_CATEGORIES = {
         "weekly_hours",
     ],
     "Wealth": [
+        "owned_land_value",
+        "property_wealth",
+        "corporate_wealth",
         "land_value",
```

Comment:
should this be `estimated_land_value`? and similarly below, `estimated_carbon_emissions`?

could be separate from this pr

Reply:
Added to the labels, rather than the variable names, is that OK?